app/views/tasks/index.html.erb

```diff
@@ -0,0 +1 @@
+<%= render partial: "form"%>
```

Comment:
Because the `_form.html.erb` partial is only used in this one location, we should just have the code from that file in this spot. Partials can be useful for helping us DRY up our view code, but only if we are actually using them in multiple places.

Additionally, "form" is probably not the ideal name for this partial as it does not contain a form but instead actually lists all of the tasks. The partial we have called "task_summary" is probably the one that should be named "form".